### DISCUSSION TITLE
Lost my account and my graduation thesis, I feel like I want to die

### DISCUSSION THREAD

**Discussion by rushlistener** [community/community#28473] [Other Feature Feedback, Questions, & Ideas]
My GitHub account was suspended with GitHub claiming unespecified violation of GitHub terms of service. 

I’m desperate because I have no way to access all my information in GitHub, all my projects, including my university thesis of which I have only a partial backup. I'm desperate because I can't work on my thesis and I will lose a giant ammount of work, I spent a lot of time in this. 

I had a lot of university projects there as well, and I used to store a lot of personal projects and I don't have a local copy anymore of a lot of these projects. 

I'm really desperate and I don't know what to do, because it feels like I lost everything. I'm feeling a lot of anxiety and crying and I'm having  a lot of physicall stress symptoms, I've been feeling really bad. 
I don't know what I'll do if I lose my thesis that I put a giant amount of work, I feel like a I really want to die because of this. 
Of having to start from scratch. 
I tried getting in contact with support, but no one answers and I can't even login and I can't even see my tickets. 

GitHub also destroyed all my public repositories of college projects, a lot of which the only copy was stored in GitHub. 

Over 6 years of work and countless hours have been destroyed, and not a single human being is available to help me understand what I did wrong so I can fix, I would do anything to be able to access all the work I stored in GitHub!

**Comment by billythekid** [comment]
Hi there!

We're not able to discuss account specifics on here so **if you've not already done** so please raise a [Reinstatement Request](https://support.github.com/contact/reinstatement)

If you have already raised this please be patient and someone will get back to you as soon as they are able with any information etc. Submitting more requests will only slow any response.

You should note that the GitHub [Terms of Service](https://docs.github.com/github/site-policy/github-terms-of-service#3-account-requirements) state that an individual may not maintain more than one free user account, even if the one you have is restricted in some way, so this additional account _may_ also find itself restricted in future. 

As I say, please exercise patience and someone will be in touch on the email address you fill in on the reinstatement form.

Thanks,
Billy 🤠

**Reply by rushlistener** [reply]
I'm completely unable to access the account. I can't even login to open support tickets. It feels like this is an experience that is designed to inflict pain and punishment to users. I'm feeling awful, I feel like a should kill my myself, I basically lost my thesis that I worked on for the past one and a half years, and also much of the work that I did in uni classes.

**Reply by airtower-luna** [reply]
You do not need to log in to file tickets for account problems. Here's the form: https://support.github.com/contact/account
